frontend/src/constants/badges/types.ts

```diff
@@ -0,0 +1 @@
+export type PackageBadge = 'experimental' | 'stable' | 'retired';
```

Comment:
Should there also be a 'deprecated' badge?

Reply:
In this context, 'retired' is taking the place of the 'deprecated' badge as it is a recognized lifecycle badge of repomountie (https://github.com/bcgov/repomountie/blob/master/doc/lifecycle-badges.md). 

If we think this should be changed to deprecated in the future it will be an easy change.